clang/test/CheckedC/static-checking/bounds-decl-checking-bsi.c

```diff
@@ -64,7 +66,9 @@ _Array_ptr<int> f14(unsigned num) {
 }
 
 _Array_ptr<int> f15(unsigned num1, unsigned num2){
-  _Array_ptr<int> p : byte_count(num1) = test_f13(num2); // expected-warning {{cannot prove declared bounds for 'p' are valid after initialization}} \
+  _Array_ptr<int> p : byte_count(num1) = test_f13(num2); // expected-error {{it is not possible to prove that the inferred bounds of 'p' imply the declared bounds of 'p' after initialization}} \
+                                                         // expected-note {{the declared upper bounds use the value of the variable 'num1', and there is no relational information between 'num1' and any of the variables used by the inferred upper bounds}} \
```

Comment:
The notes about the bounds should be displayed before the note about lack of relational information.  This will be confusing for a programmer to follow.   This can be addressed in follow-up work.

Reply:
Thanks for the suggestion @dtarditi. Created issue #912 to track this.